velodyne_pointcloud/include/velodyne_pointcloud/datacontainerbase.hpp

```diff
@@ -83,7 +83,7 @@ class DataContainerBase
     cloud.width = config_.init_width;
     cloud.height = config_.init_height;
     cloud.is_dense = static_cast<uint8_t>(config_.is_dense);
-    cloud.row_step = cloud.width * cloud.point_step;
+    cloud.row_step = init_width * cloud.point_step;
```

Comment:
I don't think this applies to the `ros2` branch.

`init_width` is passed into the constructor.  We then use `init_width` to initialize `config_.init_width` in line 63-64 above.  We then initialize `cloud.width` from `config_.init_width` in line 83.  So at this point, `cloud.width` should always be the same as `init_width`, and thus I don't think this is necessary.

Reply:
You're right. I will drop this.